CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+Contributing
+============
+
+This is a fork of `git@github.com:cucumber-ltd/shouty.java.git`
+
+In order to maintain updates from cucumber-ltd, and to support contributing back, we use GitFlow
+
```

Comment:
I just wanted to follow up and say that the Docker work is brilliant! Is there a particular reason you're suggesting GitFlow @temyers, or could we remove that?